src/main/java/com/blackducksoftware/integration/hub/alert/channel/hipchat/HipChatChannel.java

```diff
@@ -87,21 +87,22 @@ public String testGlobalConfig(final GlobalHipChatConfigEntity entity) {
         }
         final RestConnection restConnection = channelRestConnectionFactory.createUnauthenticatedRestConnection(HIP_CHAT_API);
         if (restConnection != null) {
-            final List<String> urlSegments = Arrays.asList("v2", "room", "*", "notification");
-            final Map<String, String> queryParameters = new HashMap<>();
-            queryParameters.put("auth_test", "true");
+            try {
+                final List<String> urlSegments = Arrays.asList("v2", "room", "*", "notification");
+                final Map<String, String> queryParameters = new HashMap<>();
+                queryParameters.put("auth_test", "true");
 
-            final Map<String, String> requestHeaders = new HashMap<>();
-            requestHeaders.put("Authorization", "Bearer " + entity.getApiKey());
-            requestHeaders.put("Content-Type", "application/json");
+                final Map<String, String> requestHeaders = new HashMap<>();
+                requestHeaders.put("Authorization", "Bearer " + entity.getApiKey());
+                requestHeaders.put("Content-Type", "application/json");
 
-            // The {"message":"test"} is required to avoid a BAD_REQUEST (OkHttp issue: #854)
-            final ChannelRequestHelper channelRequestHelper = new ChannelRequestHelper(restConnection);
-            final HttpUrl httpUrl = restConnection.createHttpUrl(urlSegments, queryParameters);
-            final Request request = channelRequestHelper.createMessageRequest(httpUrl, requestHeaders, "{\"message\":\"test\"}");
+                // The {"message":"test"} is required to avoid a BAD_REQUEST (OkHttp issue: #854)
+                final ChannelRequestHelper channelRequestHelper = new ChannelRequestHelper(restConnection);
+                final HttpUrl httpUrl = restConnection.createHttpUrl(urlSegments, queryParameters);
+                final Request request = channelRequestHelper.createMessageRequest(httpUrl, requestHeaders, "{\"message\":\"test\"}");
+
+                Response response = null;
 
-            Response response = null;
-            try {
```

Comment:
Something I meant to fix, but missed was the null check on _response_. I don't think it can be null any more, but if that's not true, the 'return "Invalid API key: " + response.message()' needs to have a null check as well. I believe the former is true so we should remove it.